sdks/python/gen_xlang_wrappers.py

```diff
@@ -212,9 +179,60 @@ class name. This can be overriden by manually providing a name.
     f.write(
         "# NOTE: This file is autogenerated and should "
         "not be edited by hand.\n")
+    f.write(
+        "# Configs are generated based on the expansion service\n"
+        f"# configuration in {input_services.replace(PROJECT_ROOT, '')}.\n")
+    f.write("# Refer to sdks/python/gen_xlang_wrappers.py for more info.\n")
     dt = datetime.datetime.now().date()
-    f.write(f"# Last updated on: {dt}\n\n")
+    f.write(f"#\n# Last updated on: {dt}\n\n")
     yaml.dump(transform_list, f)
+  logging.info("Successfully wrote transform configs to file: %s", output_file)
+
+
+def validate_sdks_destinations(sdk, dest, service, identifier=None):
```

Comment:
given that we now assemble external transform wrappers in one folder, is it still important to specify particular destinations (say, apache_beam/io vs apache_beam/io/gcp) when all generated wrappers will live in _external_transforms  ?

Reply:
in that case, the destination or output file could be "io_gcp.py" or "io_gcp",  as opposed to 'apache_beam/io/gcp' + later parsing out the apache_beam portion, splitting the io/gcp -> io_gcp, etc.

Reply:
I figured we can handle this part in the script to make it easier for someone making adjustments to the config. We convey where the transform would be made available to (ie. where users would import them from).

A new developer who just wants to set a destination for a service/transform may find it easier to understand the result of 
```yaml
'beam:schematransform:org.apache.beam:some_transform:v1':
  destinations:
    python: 'apache_beam/io/gcp'
```
Rather than
```yaml
'beam:schematransform:org.apache.beam:jdbc_write:v1':
  destinations:
    python: 'io_gcp'
```